app/serializers/source_data/product_serializer.rb

```diff
@@ -18,6 +18,7 @@ def deserialize(payload, product_type)
       if product_type == "ShopifyProduct"
         properties = payload.delete("attributes")
         payload["properties"] = properties
+        payload["product_category_id"] = "gid://shopify/TaxonomyCategory/#{payload["product_category_id"]}"
```

Comment:
Any reason this can't come from `Category#gid`?

Reply:
Shopify/2022-02's category nodes are not seeded